molten.c

```diff
@@ -113,7 +109,11 @@ static const zend_module_dep molten_deps[] = {
     ZEND_MOD_OPTIONAL("redis")    
     ZEND_MOD_OPTIONAL("mongodb")    
     ZEND_MOD_OPTIONAL("mysqli")    
+#ifdef HAS_PDO
+    ZEND_MOD_REQUIRED("PDO")
```

Comment:
HAS_PDO is for check "ext/pdo/php_pdo_driver.h"  at build time,  but at runtime, if he has pdo extension, he also can trace it .